examples/complete-ecs/main.tf

```diff
@@ -0,0 +1,58 @@
+provider "aws" {
+  region  = "eu-west-1"
+  version = "v1.18.0"
+}
+
+provider "terraform" {}
+
+locals {
+  name = "my-ecs"
+}
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "v1.30.0"
```

Comment:
No need to pin version here. We need to stay on the edge with examples to be able to see that this still works (yes, we lack automated tests for modules).